addOns/commonlib/src/test/java/org/zaproxy/addon/commonlib/CookieUtilsUnitTest.java

```diff
@@ -192,4 +192,37 @@ public void shouldFindAttributeEvenIfItHasDifferentCase() {
         // Then
         assertThat(found, is(equalTo(true)));
     }
+
+    @Test
+    public void shouldNotFindCookiePlusNameIfNameIsNull() {
+        // Given
+        String fullHeader = "Set-Cookie: foo; Attribute1";
+        String headerValue = "foo; Attribute1";
+        // When
+        String name = CookieUtils.getSetCookiePlusName(fullHeader, headerValue);
+        // Then
+        assertThat(name, is(equalTo(null)));
+    }
+
+    @Test
+    public void shouldKnowThatNameDoesNotIncludeSemiColon() {
+        // Given
+        String fullHeader = "Set-Cookie: foo; Attribute1; Attribute2=AV2";
```

Comment:
Was this actually found in the wild or was to exercise the potential NPE? This does not conform to the expected format.

Reply:
Just checking, I know the name would be incorrect without the semicolon check.

Reply:
It was two years ago. However, looking at the "AV2" value I'm guessing it was actually found.

Reply:
It could be. Just doesn't seem like something I would have chosen. Though maybe it's simply in use elsewhere.